src/ir/ir-builder.h

```diff
@@ -31,29 +31,56 @@ class IRBuilderWrapper: public Nan::ObjectWrap, public FromValueMixin<IRBuilderW
     template<ConvertOperationFn method>
     static NAN_METHOD(ConvertOperation);
 
+    static NAN_METHOD(CreateAdd);
     static NAN_METHOD(CreateAlloca);
     static NAN_METHOD(CreateAlignedLoad);
     static NAN_METHOD(CreateAlignedStore);
+    static NAN_METHOD(CreateAnd);
+    static NAN_METHOD(CreateAShr);
+    static NAN_METHOD(CreateBitCast);
     static NAN_METHOD(CreateBr);
     static NAN_METHOD(CreateCall);
     static NAN_METHOD(CreateCondBr);
     static NAN_METHOD(CreateExtractValue);
     static NAN_METHOD(CreateFNeg);
     static NAN_METHOD(CreateGlobalStringPtr);
     static NAN_METHOD(CreateGlobalString);
-    static NAN_METHOD(CreateInBoundsGEP);
     static NAN_METHOD(CreateInBoundsGEPWithoutType);
     static NAN_METHOD(CreateInBoundsGEPWithType);
-    static NAN_METHOD(CreateInsertValue);
+    static NAN_METHOD(CreateInBoundsGEP);
     static NAN_METHOD(CreateIntCast);
+    static NAN_METHOD(CreateInsertValue);
```

Comment:
What's the reason you declared all these methods on IRBuilder? The header only specifies functions that are defined in the implementation file. However, the C++ implementation does not define all these methods. These methods are all implemented by the template `NANBinaryOperation` and only exposed to JavaScript (you can't call these methods on the C++ implementation).

```suggestion
```